app-authz-rest-springboot/README.md

```diff
@@ -169,4 +169,4 @@ Integration test of the Quickstart
 1. Make sure you have an Keycloak server running with an admin user in the `master` realm or use the provided docker image
 2. Be sure to set the `TestHelper.keycloakBaseUrl` in the `createArchive` method (default URL is localhost:8180/auth).
 3. Set accordingly the correct url for the `keycloak.auth-server-url` in the test [application.properties](src/test/resources/application.properties).
-4. Run `mvn test`
\ No newline at end of file
+4. Run `mvn test -DskipTests=false`
```

Comment:
I find the need to specify skip tests to false is confusing and a bit redundant. If you're providing `mvn test` it is implicit that we don't want to skip these tests.